src/models/room.js

```diff
@@ -555,9 +556,9 @@ Room.prototype.clearLoadedMembersIfNeeded = async function() {
  */
 Room.prototype._cleanupAfterLeaving = function() {
     this.clearLoadedMembersIfNeeded().catch((err) => {
-        console.error(`error after clearing loaded members from ` +
+        logger.error(`error after clearing loaded members from ` +
             `room ${this.roomId} after leaving`);
-        console.dir(err);
+        logger.dir(err);
```

Comment:
does this actually work?

Reply:
@turt2live nice catch. Yes this is not working. What can we do about that? Maybe just a log

Reply:
just a log is probably fine, yea

Reply:
Done